packages/components/src/components/carousel/carousel.tsx

```diff
@@ -263,19 +273,23 @@ export class Carousel {
                 Skip carousel entries
               </PrefixedTagNames.pLinkPure>
             )}
-            <PrefixedTagNames.pButtonPure
-              {...btnProps}
-              icon="arrow-left"
-              ref={(ref) => (this.btnPrev = ref)}
-              onClick={() => slidePrev(this.splide, this.amountOfPages)}
-            />
-            <PrefixedTagNames.pButtonPure
-              {...btnProps}
-              icon="arrow-right"
-              ref={(ref) => (this.btnNext = ref)}
-              onClick={() => slideNext(this.splide, this.amountOfPages)}
-              onKeyDown={this.onNextKeyDown}
-            />
+            {this.isDraggable && (
+              <PrefixedTagNames.pButtonPure
+                {...btnProps}
+                icon="arrow-left"
+                ref={(ref) => (this.btnPrev = ref)}
+                onClick={() => slidePrev(this.splide, this.amountOfPages)}
+              />
+            )}
+            {this.isDraggable && (
```

Comment:
Could be combined with single expression like `{this.isDraggable && [JsxEl1, JsxEl2]}`.